.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,46 @@
-## Purpose
-<!--- Describe the problems, issues, or needs driving this feature/fix and include links to related issues -->
-The purpose of this PR is to fix #<issue-number>
+---
+labels: pending-review
+assignees:
+  - jayasanka-sack
+  - akshika47
+  - anjula-sack
+  - Piumal1999
+---
 
-## Goals
-<!---  Describe the solutions that this feature/fix will introduce to resolve the problems described above -->
 
-## Approach
-<!--- Describe how you are implementing the solutions. Include a link to a Markdown file or Google doc if the feature write-up is too long to paste here. -->
+To suggest a change to the [Handbook](https://handbook.sefglobal.org/), please review the [Handbook Guidlines](https://handbook.sefglobal.org/organisation/handbook/usage#handbook-guidelines) section.
 
-### Screenshots
-<!---  Include an animated GIF or screenshot if the change affects the UI.  -->
+<!-- please list any related epics, issues or PRs here. 
+	This field is optional, check this: https://handbook.sefglobal.org/organisation/handbook/usage#how-to-change-or-define-a-process -->
+### Related to:
 
-##  Checklist
-- [ ] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
-- [ ] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
-- [ ] My code follows the style guidelines of this project
-- [ ] I have performed a self-review of my own code
-- [ ] I have commented my code, particularly in hard-to-understand areas
-- [ ] I have made corresponding changes to the documentation
+### Please provide a brief explanation for this change:
+<!-- (If there's a due date for merging this PR, be sure to include it here.) -->
 
-## Related PRs
-<!--- List any other related PRs --> 
+**Preview Link:**
+<!---  This PR will be automatically deployed to surge. -->
+<!---  Once you submit the PR, replace "{PR_NUMBER}" with your PR number. ex: https://pr-11-sef-handbook.surge.sh -->
+<!--- Feel free to modify the link with the exact path -->
+https://pr-{PR_NUMBER}-sef-handbook.surge.sh/
```

Comment:
Can we provide an example of this? Most get confused and add the number with the curly brackets.
ex: https://pr-12-sef-handbook.surge.sh/

Reply:
I included it. It's already there.

```
<!---  Once you submit the PR, replace "{PR_NUMBER}" with your PR number. ex: https://pr-11-sef-handbook.surge.sh -->
```

Reply:
oh cool didn't see the comments